_config.yml

```diff
@@ -110,8 +110,8 @@ social:
 # Home Assistant release details
 current_major_version: 2024
 current_minor_version: 3
-current_patch_version: 1
-date_released: 2024-03-14
+current_patch_version: 2
+date_released: 2024-03-22
```

Comment:
Is it intentional that this date is off by one compared to the one in the changelog?

Reply:
Caused by generation date